Bottom: 01b02f57f0129c9b1ddecf761795858e57bf4a93
Top:    82ccc1598b866dd10eb26d5d64d800db4829fcb0
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-23 11:20:41 +0800

Refresh of add-a-vq-meta-data-iotlb

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index af90da1..dbd9561 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1017,10 +1017,6 @@ int vhost_process_iotlb_msg(struct vhost_dev *dev,
 		vhost_iotlb_notify_vq(dev, msg);
 		break;
 	case VHOST_IOTLB_INVALIDATE:
-		if (!dev->iotlb) {
-			ret = -EFAULT;
-			break;
-		}
 		vhost_vq_meta_reset(dev);
 		vhost_del_umem_range(dev->iotlb, msg->iova,
 				     msg->iova + msg->size - 1);
